lib/sync/installation.js

```diff
@@ -45,7 +46,15 @@ module.exports.processInstallation = (app, queues) => {
       // no more data (last page was processed of this job type)
       if (!getNextTask(subscription)) {
         subscription.set('syncStatus', 'COMPLETE')
-        app.log(`Sync status for installationId=${installationId} is complete`)
+        var message = `Sync status for installationId=${installationId} is complete`
+        if (job.data.startTime !== undefined) {
+          const endTime = new Date()
+          const timeDiff = endTime - Date.parse(job.data.startTime)
+          message = `${message} startTime=${job.data.startTime} endTime=${endTime.toJSON()} diff=${timeDiff}`
+          statsd.histogram('full_sync', timeDiff)
```

Comment:
I'm completely open to better metric names than `full_sync`

Reply:
I think it's serviceable. Do you mind putting a comment above the `statsd.histogram` call that says what `full_sync` measures? That way when someone looks at it in datadog and then goes and searches the code, they'll find something like (I dunno if the below phrasing is correct or well worded):
```js
// full_sync measures the duration from start to finish of a complete scan and sync of github issues translated to tickets
```